helpers/error_handler/error_constant.go

```diff
@@ -0,0 +1,19 @@
+package helpers
+
+const (
+	ChangeToRootDirFailure   = "Failed to change to root directory: "
+	ConnectToDatabaseFailure = "Failed to connect database: "
+	DeleteRedisJobFailure    = "Failed to delete redis job: "
+	EnqueueJobFailure        = "Failed to enqueue job: "
+	HashPasswordFailure      = "Failed to hash password: "
+	JSONMarshalFailure       = "Failed to marshal json: "
+	JSONUnmarshalFailure     = "Failed to unmarshal json: "
+	MigrateDatabaseFailure   = "Failed to migrate database: "
+	ReadResponseBodyFailure  = "Failed to read response body: "
+	RecordInitializeFailure  = "Failed to init record: "
+	RecordStopFailure        = "Failed to stop record: "
+	RequestInitializeFailure = "Failed to init request: "
+	RequestFailure           = "Failed to make a request: "
+	ScanRowFailure           = "Failed to scan row: "
+	StartOAuthServerFailute  = "Failed to setup oauth server: "
```

Comment:
```suggestion
	StartOAuthServerFailure  = "Failed to setup oauth server: "
```

Reply:
Fixed in 4057bb3668bf88d76ae2c862cbae07fc03e92ee8 06cf1b98470ef5fdeb40e1e1b3ff2f1b16a16826